packages/ai/cypress/specs/AINoticeIndicator.cy.tsx

```diff
@@ -0,0 +1,167 @@
+import AINoticeIndicator from "../../src/AINoticeIndicator.js";
+
+describe("AI Notice Indicator", () => {
+	describe("Rendering and Interaction", () => {
+		beforeEach(() => {
+			cy.mount(<AINoticeIndicator></AINoticeIndicator>);
+		});
+
+		const attributionText = "Created with AI.";
+		const verificationText = "Verify results before use.";
+
+		it('Should render the component', () => {
+			cy.get('ui5-ai-notice-indicator').should('exist');
+		});
+
+		it("Should display closed responsive poppover", () => {
+			cy.get("[ui5-ai-notice-indicator]")
+				.shadow()
+				.find("[ui5-responsive-popover]")
+				.should("exist");
+
+			cy.get("[ui5-ai-notice-indicator]")
+				.shadow()
+				.find("[ui5-responsive-popover]")
+				.should("have.prop", "open", false);
+		});
+
+		it("Should display correct properties for Default mode", () => {
+
+			cy.get("[ui5-ai-notice-indicator]")
+				.shadow()
+				.find("[ui5-link]")
+				.should('have.attr', 'icon', '')
+				.should("exist");
+
+			cy.get("[ui5-ai-notice-indicator]")
+				.shadow()
+				.find("[ui5-link]")
+				.contains(attributionText)
+				.should("exist");
+
+			cy.get("[ui5-ai-notice-indicator]")
+				.shadow()
+				.find("[ui5-label]")
+				.contains(verificationText)
+				.should("exist");
+		});
+
+		it("Should display correct properties for Emphasized mode", () => {
+			cy.get('ui5-ai-notice-indicator').invoke('attr', 'mode', 'Emphasized');
+
+			cy.get("[ui5-ai-notice-indicator]")
+				.shadow()
+				.find("[ui5-link]")
+				.should('have.attr', 'icon', 'ai')
+				.should("exist");
+
+			cy.get("[ui5-ai-notice-indicator]")
+				.shadow()
+				.find("[ui5-link]")
+				.contains(attributionText)
+				.should("exist");
+
+			cy.get("[ui5-ai-notice-indicator]")
+				.shadow()
+				.find("[ui5-label]")
+				.contains(verificationText)
+				.should("exist");
+		});
+
+		it("Should display correct properties for Shortened mode", () => {
+			cy.get('ui5-ai-notice-indicator').invoke('attr', 'mode', 'Shortened');
+
+			cy.get("[ui5-ai-notice-indicator]")
+				.shadow()
+				.find("[ui5-link]")
+				.should('have.attr', 'icon', '')
+				.should("exist");
+
+			cy.get("[ui5-ai-notice-indicator]")
+				.shadow()
+				.find("[ui5-link]")
+				.contains(attributionText)
+				.should("exist");
+
+			cy.get("[ui5-ai-notice-indicator]")
+				.shadow()
+				.find("[ui5-link]")
```

Comment:
VerificationText is not a link, it's a label with your current implementation, text after updating the rendering per the other comment